tests/boot/boot_from_pxe.pm

```diff
@@ -166,6 +166,11 @@ sub run {
     send_key 'ret';
     save_screenshot;
 
+    # If the remote repo doesn't exist, the machine will silently boot
+    # from disk.
+    die 'PXE boot failed'
```

Comment:
shouldnt the error comment be more explanatory?

Reply:
I also think the comments in L169-L170 should be dropped as the test will fail if remote repo does not exist, right?